client-app/src/examples/contactMobile/cmp/TileView.ts

```diff
@@ -0,0 +1,49 @@
+import {hoistCmp} from '@xh/hoist/core';
+import ContactsPageModel from '../ContactsPageModel';
+import {tileFrame, vbox, div, span, filler} from '@xh/hoist/cmp/layout';
+
+import {favoriteButton} from './FavoriteButton';
+import '../../contact/cmp/TileView.scss';
+
+const tileView = hoistCmp.factory<ContactsPageModel>({
+    render({model}) {
+        return tileFrame({
+            spacing: 10,
+            maxTileHeight: 200,
+            minTileHeight: 150,
+            maxTileWidth: 200,
+            minTileWidth: 150,
+            items: model.records.map(record => tile({record}))
+        });
+    }
+});
+
+const tile = hoistCmp.factory<ContactsPageModel>(({model, record}) => {
+    const {gridModel} = model,
+        isSelected = gridModel.selectedId === record.id,
+        {profilePicture, name} = record.data;
+
+    return vbox({
+        style: {backgroundImage: `url(${profilePicture})`},
+        className: `tb-contact-tile ${isSelected ? 'tb-contact-tile--selected' : ''}`,
+        items: [
+            div({
+                className: 'tb-contact-tile__bar',
+                // This inline style defintion keeps me from needing to fight with
+                // existing styles when making the mobile button want to center itself
+                style: {alignItems: 'center'},
```

Comment:
I had a fair bit of fighting the styling while working on this.

We define some styles in SCSS files to customer a component or layout. Said component / layout will already have some styling present from existing styles from XH / TB / BP. Then additionally you can define some styles on an element component which are writing direct to the styles tag. 
This ultimately meant for me that any styles I applied in an SCSS file were clobbered right off by either an existing style tag or - more often - existing styles with far greater specificity. In this case, using an inline style was of highest import, and so was allowing this align-items style to win over the native styles applied to a mobile button.